suite/tests/api/ir_aarch64.c

```diff
@@ -282,11 +282,33 @@ test_ldar(void *dc)
 }
 
 static void
-test_fmov_general(void *dc)
+test_instrs_with_logic_imm(void *dc)
 {
     byte *pc;
     instr_t *instr;
 
+    instr = INSTR_CREATE_and(dc, opnd_create_reg(DR_REG_X10),
+                             opnd_create_reg(DR_REG_X9), OPND_CREATE_INT(0xFFFF));
```

Comment:
(There's an issue on auto-generating the INSTR_CREATE_ macros -- and presumably the tests as well.)

Reply:
(y) I'll check with @AssadHashmi, not sure how far he got so far. At least for the remaining NEON instructions, I try to generate most things. Except for some instructions  with uncommon operands, like fmov.